### ISSUE TITLE
[FEATURE] - Can bookmark support sorting and grouping?

### ISSUE THREAD

**Issue by bruce-wayne-business** [alefragnani/vscode-bookmarks#709]
Can you provide a sorting and grouping function for bookmarks? Sometimes when marking code, I need to classify different functional modules, and sometimes I also need to mark the execution order of the code.

**Comment by phazei** [comment]
I actually just came here to request something similar.  Being able to rearrange the bookmarks in the list would be good, I went to sort them into order of importance and realized there was no way to do that.

I was going to also suggest multiple colors for the bookmarks, but I suppose that ties into grouping.  If they can be dragged around then they could be dragged to different groups, and each group could have a optionable color.  There could be a group marked active that all new bookmarks go into.  Then if someone switches tasks, they can change the active group and new bookmarks will go into that group.
